src/Outsystems.SetupTools/Functions/Install-OSIntegrationStudio.ps1

```diff
@@ -0,0 +1,250 @@
+function Install-OSIntegrationStudio
+{
+    <#
+    .SYNOPSIS
+    Installs or updates the OutSystems Integration Studio.
+    This script should be used for installing Integration Studio with version "11.14.17.61" and higher.
+    Do not use this script to install or update Integration Studio packaged with Development Environment.
+
+    .DESCRIPTION
+    This will installs or updates the OutSystems Integration Studio.
+    if the Integration Studio is already installed it will check if version to be installed is higher than the current one and update it.
+
+    .PARAMETER InstallDir
+    Where the Integration Studio will be installed. If the Integration Studio is already installed, this parameter has no effect.
+    If not specified will default to %ProgramFiles%\Outsystems
+
+    .PARAMETER SourcePath
+    If specified, the function will use the sources in that path. if not specified it will download the sources from the OutSystems repository.
```

Comment:
```suggestion
    If specified, the function will use the sources in that path. If not specified it will download the sources from the OutSystems repository.
```